pkg/controller/bootstrap/util.go

```diff
@@ -17,36 +17,40 @@ limitations under the License.
 package bootstrap
 
 import (
+	"fmt"
 	"time"
 
-	"k8s.io/klog/v2"
+	"golang.org/x/net/context"
 
 	"k8s.io/api/core/v1"
 	bootstrapapi "k8s.io/cluster-bootstrap/token/api"
 	bootstrapsecretutil "k8s.io/cluster-bootstrap/util/secrets"
+	"k8s.io/klog/v2"
 )
 
-func validateSecretForSigning(secret *v1.Secret) (tokenID, tokenSecret string, ok bool) {
+func validateSecretForSigning(ctx context.Context, secret *v1.Secret) (tokenID, tokenSecret string, ok bool) {
+	logger := klog.FromContext(ctx)
 	nameTokenID, ok := bootstrapsecretutil.ParseName(secret.Name)
 	if !ok {
-		klog.V(3).Infof("Invalid secret name: %s. Must be of form %s<secret-id>.", secret.Name, bootstrapapi.BootstrapTokenSecretPrefix)
+		form := fmt.Sprintf("%s<secret-id>", bootstrapapi.BootstrapTokenSecretPrefix)
+		logger.V(3).Info("Invalid secret name. Must be of form %s<secret-id>.", "secretName", secret.Name, "form", form)
```

Comment:
```suggestion
		logger.V(3).Info("Invalid secret name. Must be the right form", "secretName", secret.Name, "form", form)
```

Reply:
My bad. I left it out to delete it. Thanks!